### ISSUE TITLE
Can't get "-e VIRT_BRANCH=stable/7" to work on Windows

### ISSUE THREAD

**Issue by fgiasson** [nvdk/virtuoso-rpm-builder#1]
Hi, 

I was wondering if you tried to run `virtuoso-rpm-builder` on Windows? If I try via the docker commandline, then it always appears to ignore the `-e`. If I try to set the environment variable in Kitematic, and to run it from there, same result. I always see it using the latest development branch instead of what is specified in `VIRT_BRANCH`. Looking at the code and everything should be allright, but it is unfortunately not :(

Thanks!

**Comment by nvdk** [comment]
I reproduced the issue, I'll try to look into it later this week.

**Comment by fgiasson** [comment]
Hi @nvdk!

Wonderful thanks. For now, I had to recreate an image after changing the variable directly in the code. It works once I removed the `\r` from the files so I am good, but lack the initial intended flexibility :)

Otherwise great work with this! I will try to find the time to do the same to create DEB files for Virtuoso. This will be part of the version 3.4 of the OSF Installer:
- https://github.com/structureddynamics/Open-Semantic-Framework-Installer/tree/3.4
